pom.xml

```diff
@@ -44,7 +44,7 @@
         <emrapiVersion>1.18</emrapiVersion>
         <calculationVersion>1.2</calculationVersion>
         <providerManagementVersion>2.5.0</providerManagementVersion>
-        <registrationcoreVersion>1.6</registrationcoreVersion>
+        <registrationcoreVersion>1.7-SNAPSHOT</registrationcoreVersion>
```

Comment:
Do you still need this change? Now that there is no longer a change in the registration-core module?